README.md

```diff
@@ -24,20 +26,28 @@ Data collection is easy. Decision making is hard. H2O makes it fast and easy to
 <a name="Downloading"></a>
 ## 1. Downloading H2O-3
 
-While most of this README is written for developers who do their own builds, most H2O users just download and use a pre-built version.  If that's you, just follow these steps:
+While most of this README is written for developers who do their own builds, most H2O users just download and use a pre-built version.  If you are a Python or R user, the easiest way to install H2O is via [PyPI](https://pypi.python.org/pypi/h2o) or [Anaconda](https://anaconda.org/anaconda/h2o) (for Python) or [CRAN](https://CRAN.R-project.org/package=h2o) (for R):  
+
+#### Python
+
+```bash
+pip install h2o
+```
+
+#### R
+
+```r
+install.packages("h2o")
+```
 
-1.  Point to <http://h2o.ai>
-2.  Click on Download
-3.  Click on Download H2O-3
-4.  Choose a platform
-5.  Follow the instructions on the download page
+For the latest stable, nightly, Hadoop (or Spark / Sparkling Water) releases, or the stand-alone H2O jar, please visit: [https://h2o.ai/download](https://h2o.ai/download)
 
-For documentation, please visit <http://docs.h2o.ai>.
+More info on downloading & installing H2O is available in the [H2O User Guide](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html).
```

Comment:
No change required. Just a note that the downloading.html link will look like the following after 3.10.5.1:
http://h2o-release.s3.amazonaws.com/h2o/master/3856/docs-website/h2o-docs/downloading.html